packages/react-components/index.js

```diff
@@ -0,0 +1,88 @@
+import AcceptTermsPrompt from './AcceptTermsPrompt';
```

Comment:
This index file exports all of our built components so that `core` has easy access using:

```js
export * from '@department-of-veterans-affairs/react-components';
```

Also made possible by the `files` property in the `package.json`

Reply:
Oof. This should really be part of the build process. That is, we should have a `src/index.js` and use a proper bundler. But hey. Future enhancements. :man_shrugging: I know that's kind of out of scope for this PR.

Reply:
Yeah... our options were to rework the build process and standardize it, possibly combining the react components and web components into the same package at the root level, or keep things separate and go with a monorepo to combine them, and the second option seemed like a lighter touch. I definitely don't _love_ the current build process for `react-components`, but it seems easier to setup a monorepo and to deal with that build later.